### PR TITLE
adding ability to exclude_metadata and exclude_git_data, and exit 1 if failed

### PR DESCRIPTION
**Pull request by joshjohanning**
- Adding ability to:
    + exclude_metadata
    + exclude_git_data
- exit 1 if failed - otherwise it just reports back "failed" every 15s forever 😄 

Also, updating the readme to show how you can pass in more than 1 repo.

### CODE REVIEW

**Comment by joshjohanning** [comment]
@andyfeller I also added an `IF` condition to `exit 1` if in the loop it finds export status = `failed`

File: README.md [hunk 1]
```diff
@@ -27,14 +27,17 @@ $ gh repo-export --help
...
-  gh-repo-export [options] <organization> <repo> ...
+  gh-repo-export [options] <organization> <repo1> <repo2> ...
 
 FLAGS
   -d, --debug                         Enable debugging
       --exclude-attachments           Indicates attachments should be excluded from the migration
       --exclude-releases              Indicates releases should be excluded from the migration
       --exclude-owner-projects        Indicates projects owned by the organization or users should be excluded from the migration
+      --exclude-metadata              Indicates metadata should be excluded from the migration
+      --exclude-git-data              Indicates git data should be excluded from the migration
```
Comment (andyfeller) [inline README.md]: @joshjohanning : would you be opposed to changes necessary to keep flags alphabetical?

```suggestion
      --exclude-git-data              Indicates git data should be excluded from the migration
      --exclude-metadata              Indicates metadata should be excluded from the migration
      --exclude-owner-projects        Indicates projects owned by the organization or users should be excluded from the migration
      --exclude-releases              Indicates releases should be excluded from the migration
```

I know this involves some code changes too, but I firmly believe this is important to product experience.

File: README.md [hunk 2]
```diff
@@ -27,14 +27,17 @@ $ gh repo-export --help
...
       --hostname=string               Hostname of the GitHub instance to authenticate with
+      --pat=string                    PAT to authenticate with
```
Comment (andyfeller) [inline README.md]: Can you help me understand why this as a flag versus using the [GH CLI `GH_TOKEN` or `GITHUB_TOKEN` env var](https://cli.github.com/manual/gh_help_environment)?

**Review by andyfeller — CHANGES_REQUESTED** [review]
Thank you for opening your 1st PR! ❤️ 

1 small nit request and 1 question I'd love to figure out with you, @joshjohanning

File: README.md (hunk 2 above)
Comment (joshjohanning) [reply README.md]: Oops! That was a holdover from something else I was trying and shouldn't have made it in here :)

**Comment by joshjohanning** [comment]
> Thank you for opening your 1st PR! ❤️
> 
> 1 small nit request and 1 question I'd love to figure out with you, @joshjohanning

@andyfeller totally agree! Good catches 😄. Just pushed the changes.  👀  again?

**Review by andyfeller — APPROVED** [review]
✨ thanks, @joshjohanning ! 🙇

**Pull request merged**